pkg/couchdb/couchdb.go

```diff
@@ -25,6 +25,10 @@ import (
 // a upperbound for string.
 const MaxString = mango.MaxString
 
+// This constant is used to evaluate permissions in the `Valid` method of a
+// JSONDoc object.
+const selectorReferencedBy = "referenced_by"
```

Comment:
It is duplicated between `consts` and `couchdb`. Can we define it in one package and just reference it from the other?